Modules/_decimal/_decimal.c

```diff
@@ -4017,6 +4017,27 @@ _decimal_Decimal_to_integral_exact_impl(PyObject *self, PyObject *rounding,
     return result;
 }
 
+/*[clinic input]
+_decimal.Decimal.to_integral_value
```

Comment:
Can you swap things around so that the main implementation is paired with the docstring? Then `_decimal.Decimal.to_integral` can be a very short block that calls `_decimal_Decimal_to_integral_value_impl()`.